src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -245,3 +242,11 @@ else
     </table>
 }
 
+@if (Model.Status.HasFlag(NoteStatus.Draft) || Model.HasBeenReturned)
```

Comment:
@GuidoWorkDev 

Looking at the views I think this button should possibly be added to EditDraftTransfer.cshtml as this is the only place you should submit the button from.

Reply:
@GuidoWorkDev if you have issues with the button posting to the action you will need to wrap the button in a form...